src/sandstorm/backup.c++

```diff
@@ -106,9 +106,6 @@ void BackupMain::bind(kj::StringPtr src, kj::StringPtr dst, unsigned long flags)
 }
 
 bool BackupMain::run(kj::StringPtr grainDir) {
-  KJ_ASSERT(geteuid() != 0, "can't run as user ID zero (root)");
```

Comment:
We need to remove the asserts because, for non-root installs, the perceived euid inside the outer container is zero, because that's what the user namespace maps the current user to. Without this change, backups would break for non-root installs.